.github/workflows/rust_test_tube.yml

```diff
@@ -27,6 +31,15 @@ jobs:
       - name: Build ${{ inputs.contract }}
         run: cargo test-tube-build
         working-directory: smart-contracts/contracts/${{ inputs.contract }}
+      // OPTIONAL DEPENDENCIES
```

Comment:
What's this line?? @lubkoll

Reply:
a comment:) As apparently it was not helpful, I removed it.